kuma/static/styles/components/wiki/promos.scss

```diff
@@ -201,8 +215,10 @@
     }
 
     #wiki-promo-sauce-labs-c {
-        @include bidi-style(background-position, bottom center, background-position, bottom center);
-        @include bidi-style(padding, 0 0 175px, padding, 0 0 175px);
+        @include bidi((
+            (background-position, bottom center, background-position, bottom center),
+            (padding, 0 0 175px, padding, 0 0 175px)
```

Comment:
Nit: Also here. This is the third instance so far so, I am starting to think there is a reason why it is done this way?